packages/ts/react-signals/src/Signals.ts

```diff
@@ -47,6 +47,9 @@ export abstract class ValueSignal<T> extends Signal<T> {
    */
   override set value(value: T) {
     const id = crypto.randomUUID();
+    // first set the local value:
```

Comment:
Could you describe in this comment why do we need it?

Reply:
This was more of reminder about the order of the operations, so it is not changed to first publish the value to the server and then set the local value by accident (in the future), as I couldn't think of a clear way of verifying that through a test.

Reply:
Added the reason in the comment.